src/gridcoin/contract/contract.cpp

```diff
@@ -856,6 +861,9 @@ ContractPayload Contract::Body::ConvertFromLegacy(const ContractType type) const
             // Claims can only exist in a coinbase transaction and have no
             // legacy representation as a contract:
             assert(false && "Attempted to convert legacy claim contract.");
+        case ContractType::MRC:
+            // MRCs have no legacy representation as a contract.
+            assert(false && "Attempted to convert non-existent legacy MRC contract.");
```

Comment:
I think you want a break here or it will fall through when built with `-DNDEBUG`

Reply:
We don't allow builds with NDEBUG.

Reply:
Technically I suppose break should be after each one, but in this particular switch, there is either a return statement or assert, so the break would never get executed.